packages/time-series/lib/commands/MGET.ts

```diff
@@ -18,14 +19,47 @@ export function pushFilterArgument(args: CommandArguments, filter: RedisVariadic
   return pushVariadicArguments(args, filter);
 }
 
+export type MGetRawReply2 = Array<[
+  key: string,
```

Comment:
`BlobStringReply`?

Reply:
yes, it should be BlobStringReply, this was just a port over from v2, but it's a RedisModule_ReplyWithStringBuffer() so BlobString.  Will fix.

Reply:
addressed above